### ISSUE TITLE
[Feature Request] Add deathsprites for powerup states 2 and up

### ISSUE THREAD

**Issue by Superbloxen** [Wohlstand/TheXTech#448]
When a player dies from obstacles such as crushing, lava, or falling into bottomless pits, they should display a special death sprite showing what powerup state they had before they died. This feature could be toggled using a compat.ini flag, so that old sprites wouldn't break. I made some deathsprites for the base 1.3 sprites here:
[Deathsprites.zip](https://github.com/Wohlstand/TheXTech/files/10291537/Deathsprites.zip)

**Comment by Wohlstand** [comment]
Good idea, I think, it's possible to make it work in a condition when a new sprites provided at assets, otherwise, fall back to the old one.
